source/features/prevent-link-loss.tsx

```diff
@@ -20,16 +20,25 @@ import createBanner from '../github-helpers/banner.js';
 const documentation = 'https://github.com/refined-github/refined-github/wiki/Extended-feature-descriptions#prevent-link-loss';
 
 function handleButtonClick({currentTarget: fixButton}: React.MouseEvent<HTMLButtonElement>): void {
-	/* There's only one rich-text editor even when multiple fields are visible; the class targets it #4678 */
-	const field = fixButton.form!.querySelector('textarea.js-comment-field')!;
+	let field;
+
+	if (fixButton.form) {
+		/* There's only one rich-text editor even when multiple fields are visible; the class targets it #4678 */
+		field = fixButton.form!.querySelector('textarea.js-comment-field')!;
+	} else {
+		// New commit input field
+		field = fixButton.closest('[data-testid="markdown-editor-comment-composer"]')!
+			.querySelector('textarea[aria-labelledby="comment-composer-heading"]')!;
+	}
+
```

Comment:
You can use `fixButton.closest(['form', 'test id selector']).querySelector( fieldSelector)`. The form/testid array should also be on multiple lines and commented as usual